run.py

```diff
@@ -11,8 +12,11 @@
 GPT_CALL_OUT_ON = True   #To save costs, set to False when debugging or developing
 PRINT_AS_YOU_GO = False
 
+# dotenv_path = join(dirname(__file__), '.env')
+load_dotenv()
+
 # Set up your OpenAI API key
-openai.api_key = "YOUR-API-KEY-HERE"
+openai.api_key = os.environ.get("OPENAI_API_KEY")
```

Comment:
Yes!

Reply:
Then yes you can access .env variables like any other OS specific environment variable